### DISCUSSION TITLE
Welcome to Weasis Discussions!

### DISCUSSION THREAD

**Discussion by nroduit** [nroduit/Weasis#231] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dev45321** [comment]
Hello,
i was able to build the weasis 3.8.1, and i would like to invoke the weasis through the browser link 
start weasis://%24dicom%3Aget%20-w%20https%3A%2F%2Fnroduit.github.io%2Fsamples%2FLumbar%2Fmf.xml

but i see there are no registries getting created when i run the 64bit installer post building, 

however i see code present in the weasis main.wxs file, 
the build log is clean, and no errors were seen, 
can i know if there is any solution to implement the registries while installing?

[log.txt](https://github.com/nroduit/Weasis/files/8787035/log.txt)

**Comment by nroduit** [comment]
See this [reply](https://groups.google.com/g/dcm4che/c/L0BCKZix2Oo/m/47NaRCvpAAAJ).

This is the [Windows configuration](https://github.com/actions/virtual-environments/blob/main/images/win/Windows2022-Readme.md) on Github Actions.

**Comment by twiggmitsue-art** [comment]
I'm trying to open a file in dicom, and the sistem is asking me about a password. I didn't received this cod. Where can I find it.

**Reply by nroduit** [reply]
Please note that we do not provide any assistance for third-party integrations (whether commercial or non-commercial). For questions related to integrating Weasis with a specific system or data source, please reach out to the party that provides the data or the integration itself, as they are best positioned to support their own setup.
